py3status/modules/netdata.py

```diff
@@ -18,14 +18,22 @@
 @author Shahin Azad <ishahinism at Gmail>
 """
 
+import os
 import subprocess
 
 
 class GetData:
     """Get system status.
     """
     def __init__(self, nic):
```

Comment:
Also this is the wrong place.  `self.nic` will always be `None` you need to use `post_config_hook()` as data will have been set then.  `__init__()` happens before data is initialized.